### PR TITLE
#164798278 Upgrade Authors Haven application from django 1.1 to django 2.1

### PR DESCRIPTION
**Pull request by patrickf949**
#### What does this PR do?
Upgrades authors haven from django 1.1 to django 2.1 

#### Description of Task to be completed?

- Update to Django 2 URL format.
- Update Project dependencies.
- Remove reference to non-existent Authentication backends.

#### How should this be manually tested?

After cloning the repo, cd into it and run the following commands to activate the virtual environment
- `$ virtualenv venv`
- `$ source venv/bin/activate`

Install all requirements
- `$ pip install -r requirements.txt`

Make migrations
- `$ python manage.py makemigrations`
- `$ python manage.py migrate`

Run the server 
- `$ python manage.py runserver`

#### Any background context you want to provide?
Many features in Django v1.1 - the initial django version in which author's haven is written, are no longer supported, which is why we are upgrading to version 2.1 - the latest version

#### What are the relevant pivotal tracker stories?
[#164798278](https://www.pivotaltracker.com/story/show/164798278)

### CODE REVIEW

**Review by anyatibrian — COMMENTED** [review]
its great work keep it up, though the urls are missing the base name ie path('user/', UserRetrieveUpdateAPIView.as_view(), name="retrive-update")

File: authors/settings.py [hunk 1]
```diff
@@ -137,7 +136,7 @@
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
-    'DEFAULT_AUTHENTICATION_CLASSES': (
```
Comment (e-ian) [inline authors/settings.py:140]: Hi Patrick. Well done so far. Why do you have the "DEFAULT_AUTHENTICATION_CLASSES" commented out?
If there is a valid reason for doing so. Its not proper to push commented out code to the remote.

Comment (patrickf949) [reply authors/settings.py:140]: Thanks @e-ian, I've removed the unnecessary code as advised

**Comment by patrickf949** [comment]
> its great work keep it up, though the urls are missing the base name ie path('user/', UserRetrieveUpdateAPIView.as_view(), name="retrive-update")

thanks @anyatibrian,
I've added the base names for the urls, as well as the app name

**Comment by Genza999** [comment]
Great work Patrick.

**Comment by Genza999** [comment]
Hey @patrickf949, good work. Would you kindly change the PR commit message to include a title, a body and a footer?

**Comment by anyatibrian** [comment]
great, work

**Review by anyatibrian — COMMENTED** [review]
great work

**Pull request merged**